services/ux-backend/handlers/onboarding_tokens.go

```diff
@@ -0,0 +1,110 @@
+package handler
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/red-hat-storage/ocs-operator/v4/services/types"
+)
+
+func OnboardingTokensHandler(w http.ResponseWriter, r *http.Request) {
+
+	switch r.Method {
+	case "POST":
+		onboardingToken, err := getOnboardingToken()
+		if err != nil {
+			http.Error(w, encodeResponse(types.HTTPResponse{Code: http.StatusMethodNotAllowed, Message: onboardingToken}), http.StatusInternalServerError)
+			return
+		}
+
+		onboarding := struct {
+			Token string `json:"token"`
+		}{}
+
+		onboarding.Token = onboardingToken
+		marshaledOnboardingToken, err := json.Marshal(onboarding)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		//nolint
+		w.Write(marshaledOnboardingToken)
+
+	default:
+		http.Error(w, encodeResponse(types.HTTPResponse{Code: http.StatusMethodNotAllowed, Message: "Only POST request is allowed"}), http.StatusMethodNotAllowed)
+		return
+	}
+}
+
+func getOnboardingToken() (string, error) {
+
+	tokenLifetimeInHours, err := strconv.Atoi(os.Getenv("ONBOARDING_TOKEN_LIFETIME"))
+	if err != nil {
```

Comment:
could you pls also add minor checks? something like 0 < hr <= 48, we could use negative for lifetime but that might be a security issue if unused.